### ISSUE TITLE
About training process

### ISSUE THREAD

**Issue by Youqi970124** [junyanz/pytorch-CycleGAN-and-pix2pix#1293]
When we train, how do we know whether to train the generator or the discriminator first in an epoch, and each epoch will train G and D? I mean maybe train G and D but G only train one epoch, but D train 5 epochs.

Because I didn’t see the relevant description in the program
If someone finds it, please tell me.
Very thanks.

**Comment by junyanz** [comment]
The code does not support different numbers of training iterations for G and D. You can add FOR loop for Ds for pix2pix ([here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/pix2pix_model.py#L119)) and CycleGAN ([here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/cycle_gan_model.py#L190)). People also find that simply using a larger learning rate for D ([TTUR](https://arxiv.org/abs/1706.08500)) is a simpler solution.
